aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +31,105 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
+            include different actions a user can perform. Icons can represent
+            different types of functionality. It is important that the context
+            of an icon is clear. In some cases, it may be beneficial to pair
+            icons with text to ensure the purpose of the icon is clear.
+          </SubsectionText>
+        </Subsection>
+        <ButtonRow>
+          <Button
+            label="See in Figma"
+            href="https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons"
+            primary
+            target="_blank"
+            rel="noreferrer noopener"
+          />
+        </ButtonRow>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Developing with Grommet Icons" level={3}>
+          <SubsectionText>
+            For instructions on how to install and use grommet icons within your
+            application, check out the Grommet Icons site.
+          </SubsectionText>
+        </Subsection>
+        <Button
+          label="HPE Theme Grommet Icons"
+          href="https://icons.grommet.io/?theme=hpe"
+          target="_blank"
+          rel="noreferrer noopener"
+        />
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Accessibility" level={3}>
+          <SubsectionText>
+            For screen reader accessibility there is an aria-label provided for
+            each icon. The color of the icons should also pass any color
+            contrast ratio.
+          </SubsectionText>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Icon within components" level={3}>
+          <SubsectionText>
+            Icons can be used in a variety of different ways. They are used
+            within other components such as{' '}
+            <GrommetAnchor label="Search" href={nameToPath('Search')} />
```

Comment:
Missing some spaces after the anchors. Also, I don't think we have to import Anchors as GrommetAnchors in this file -- we only had to do that in the Anchor page since the page component was also called Anchor.

Just `Anchor` should be fine for this whole page.